src/app/staking/[addressOrENSName]/deposits/DepositListAction.tsx

```diff
@@ -1,15 +1,17 @@
+"use client";
+
 import { useDepositorTotalStaked } from "@/hooks/useDepositorTotalStaked";
 import Tenant from "@/lib/tenant/tenant";
 import Link from "next/link";
 import TokenAmountDisplay from "@/components/shared/TokenAmountDisplay";
 import React from "react";
 import { useTokenBalance } from "@/hooks/useTokenBalance";
 
-interface DepositListActionProps {
+interface Props {
   address: string;
 }
 
-export const DepositListAction = ({ address }: DepositListActionProps) => {
+export const DepositListAction = ({ address }: Props) => {
```

Comment:
very pedantic comment but I like to inline the props, especially if theres only a single prop. This way as a dev you can read the component, read the props, read the types, without having to then find that interface to understand the props. It's not hard to do but I find that micro-frictions add up. Super pedantic comment though, you don't have to fix